hi-ml/src/health_ml/experiment_config.py

```diff
@@ -27,3 +37,10 @@ class ExperimentConfig(param.Parameterized):
                             doc="The Conda environment file that should be used when submitting the present run to "
                                 "AzureML. If not specified, the environment file in the current folder or one of its "
                                 "parents will be used.")
+    debug_ddp: str = param.ClassSelector(default=DebugDDPOptions.OFF, class_=DebugDDPOptions,
```

Comment:
for the typing error below:
```suggestion
    debug_ddp: DebugDDPOptions = param.ClassSelector(default=DebugDDPOptions.OFF, class_=DebugDDPOptions,
```